client/src/components/SearchResultListItemImage/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./SearchResultListItemImage";
```

Comment:
I noticed that you have a lot of index.js files that only export something. Are these files really necessary? Can't you just do the import from "./SearchResultListItemImage" directly?

Reply:
To be honest, I do not really know the reason why, I just saw how Leon did it in his component during the speed run with the logs project, and that's why I do It like that. But I will ask him tomorrow and tell you if there is a good reason to do It!